deps/hdr_histogram/Makefile

```diff
@@ -2,7 +2,7 @@ STD=
 WARN= -Wall
 OPT= -Os
 
-R_CFLAGS= $(STD) $(WARN) $(OPT) $(DEBUG) $(CFLAGS)
+R_CFLAGS= $(STD) $(WARN) $(OPT) $(DEBUG) $(CFLAGS) -DHDR_MALLOC_INCLUDE=\"hdr_redis_malloc.h\"
```

Comment:
The change in the C file won't be dropped because it's now part of the upstream (https://github.com/HdrHistogram/HdrHistogram_c/pull/89) and the change here shouldn't be dropped because this is our custom Makefile (it's not taken from anywhere).
I think the idea behind this: https://github.com/HdrHistogram/HdrHistogram_c/pull/89 was that it's the user's responsibility to define this.